cairo/ethereum/cancun/vm/instructions.cairo

```diff
@@ -169,12 +177,16 @@ func op_implementation{
     tempvar offset = opcode * 3 + 1;
 
     // Prepare arguments
+    // MARK: args assignment
```

Comment:
What's this ?

Reply:
a mark to easily find manual argument assignment so that if we can find all places where we have low-level assignments and modify them when required (otherwise you play hide-and-seek to locate all of them)